scout/server/blueprints/variants/controllers.py

```diff
@@ -84,6 +84,25 @@ def variants(store, institute_obj, case_obj, variants_query, variant_count, page
             )
 
         variant_obj["clinical_assessments"] = get_manual_assessments(clinical_var_obj)
+
+        if case_obj.get("group"):
```

Comment:
I got a bit lost here. What is this code doing? Could we add a comment?

Reply:
Good to know - then we need a new function and a docstring! See if the new version reads easier?

Reply:
Yes, I like it!